test/server/gateways/postgres/plugin.test.ts

```diff
@@ -32,6 +32,10 @@ describe("Postgres Gateway Tests", function () {
       });
     });
 
+    afterEach(function (done) {
```

Comment:
**Comment**: Why is this `afterEach` block even here?

Reply:
Removed after each block.